src/components/common/card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Image from 'next/image';
+import { useMediaLayout } from 'use-media';
+
+import 'src/styles/Card.css';
+
+type Props = {
+  imageUrl: string;
+  cardName: string;
+};
+
+const Card = ({ imageUrl, cardName }: Props) => {
+  const isWide = useMediaLayout({ minWidth: '1000px' });
+
+  return isWide === true ? (
```

Comment:
[imo]
`isWide` はbooleanなので、 `=== true` は不要ですね。

```suggestion
  return isWide ? (
```

<img width="386" alt="image" src="https://user-images.githubusercontent.com/12077457/171538929-707111d9-01e8-4ad4-817e-31edc04ccb18.png">

Reply:
私がこのコードをもう少し整理するなら、こんな感じにするかなと思いました。

このコードの意図は
* 分岐させたいところは、分岐を明示する
* 分岐の必要がない部分は、共通化する

というイメージです。

```
 // component
  const width = isWide ? 244 : 200
  const height = isWide ? 244 : 200
  const classDevice = isWide ? 'pc' : 'sp'

  return (
    <div className={`card ${classDevice}`}>
      <Image width={width} height={height} src={imageUrl} alt="Image" />
      <p>{cardName}</p>
    </div>
  );

```


```
// css
.card {
  border-radius: 8px;
  overflow: hidden;
  background-color: white;
}

.card.pc {
  width: 244px;
}

.card.mobile {
  width: 200px;
}

.card.pc p {
  font-size: 18px;
}

.card.mobile p {
  font-size: 12px;
}

```